cmake/morpheus_utils/python/python_module_tools.cmake

```diff
@@ -17,21 +17,41 @@ include_guard(GLOBAL)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
 
-include(python_config_macros)
-morpheus_utils_ensure_sk_build()
-morpheus_utils_python_modules_ensure_pybind11()
-morpheus_utils_python_modules_ensure_cython()
-
 #[=======================================================================[
 @brief : Creates an artifact set used to build a python wheel
 ex. morpheus_utils_create_python_package(package_name)
 results --
 morpheus_utils_create_python_package <PACKAGE_NAME>
 #]=======================================================================]
 function(morpheus_utils_create_python_package PACKAGE_NAME)
-
   list(APPEND CMAKE_MESSAGE_CONTEXT "package-${PACKAGE_NAME}")
 
+  morpheus_utils_python_modules_ensure_loaded()
+
+  set(prefix F_ARGV)
+  set(flags "")
+  set(singleValues SOURCE_DIRECTORY PROJECT_DIRECTORY)
+  set(multiValues "")
+
+  include(CMakeParseArguments)
+  cmake_parse_arguments(
+      "${prefix}"
+      "${flags}"
+      "${singleValues}"
+      "${multiValues}"
+      ${ARGN}
+  )
+
+  set(src_dir ".")
+  if(F_ARGV_SOURCE_DIRECTORY)
+    set(src_dir "${F_ARGV_SOURCE_DIRECTORY}")
+  endif()
+
+  set(project_dir ".")
```

Comment:
We can probably default this to the current `PACKAGE_NAME` value. i.e. the Morpheus package name is `morpheus` and the MRC package name is `mrc` which matches the package directory.

Reply:
Updated.